pkg/auth/providers/ldap/ldap_provider.go

```diff
@@ -82,6 +83,10 @@ func GetLDAPConfig(authProvider common.AuthProvider) (*v3.LdapConfig, *x509.Cert
 	return ldapProvider.getLDAPConfig()
 }
 
+func IsNotConfigured(err error) bool {
```

Comment:
can we rename this to indicate Ldap search is not configured for saml?

Reply:
it could work for both, we just don't care in the non saml search thing. im a bit against renaming as we could add this same error into more parts of getLdapConfig